src/client/utils/testData/possibleVariationsMock.ts

```diff
@@ -6,10 +6,297 @@ import {
   ApartmentType,
   BundledQuote,
   QuoteBundle,
+  SwedishApartmentQuoteDetails,
+  SwedishAccidentDetails,
+  NorwegianHomeContentsDetails,
+  NorwegianHomeContentsType,
+  DanishHomeContentsDetails,
+  DanishHomeContentsType,
 } from 'data/graphql'
 
 import { perilsMock } from './offerDataMock'
 
+export const possibleVariationsApartmentSE: QuoteBundleVariant[] = [
```

Comment:
Feels like this is gonna get out of hand quickly 😅 but I don't really know a better solution

Reply:
Right?! I fell the same. That just have cost me a lot of time. Previously, we're passing `offerData` to `Checkout` which is a smaller portion of data when compared with the solution now. Although, I have the feeling that even `offerData` has more information than what `Checkout` cares about. With the `UpsellCard`, `Checkout` needs to now about all the `possibleVariants` and `selectedBundleVariant` so we can choose between display `UpsellCard` or not. Without any further investigation, I think the ideal world would be something like providing a list of quote ids to `Checkout` and them, `Checkout` would make a query to get just the information it needs to populate it properly. I would like to implement that but I think it's a major refactor and since this is an experimental feature it kind of feels its not the right moment.